backend/dal/Helpers/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -47,6 +47,7 @@ public static IServiceCollection AddPimsDalRepositories(this IServiceCollection
             repositories.AddScoped<Repositories.IDocumentTypeRepository, Repositories.DocumentTypeRepository>();
             repositories.AddScoped<Repositories.INoteRepository, Repositories.NoteRepository>();
             repositories.AddScoped<Repositories.IEntityNoteRepository, Repositories.EntityNoteRepository>();
+            repositories.AddScoped<Repositories.IDocumentActivityRepository, Repositories.DocumentActivityRepository>();
```

Comment:
Will we have any issues from mayan if multiple users are using different instances of this repository concurrently? ie. user a gets a token using our credentials, makes a request. User b gets a different token using the same credentials and makes another request. Any issues that you can forsee?

Reply:
I don't think there will be problems with the credentials given the repository that handles that is instantiated as a singleton so will live for the duration of the pod. 
```
services.AddSingleton<IEdmsAuthRepository, MayanAuthRepository>();
```

For multiple pods I don't believe there will be problems since it returns the same token. Not sure when the token is revoked, but my Postman tests use the same token I got weeks ago